website/docs/r/notification_channel.html.markdown

```diff
@@ -74,7 +75,7 @@ Each notification channel type supports a specific set of arguments for the `pro
 In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the channel.
-
+* `status` - The status of the channel can indicate a configuration issue. The default status is DEFUALT.
```

Comment:
This is not an outcome when you create a channel, only the id is an outcome:

<img width="739" alt="Screen Shot 2022-09-05 at 12 21 55" src="https://user-images.githubusercontent.com/69894591/188415645-efc66425-122c-4431-9250-c8e8af68e16e.png">
---
The status is a computed attribute in the schema and you're seeing it in the plan:

<img width="360" alt="Screen Shot 2022-09-05 at 12 24 42" src="https://user-images.githubusercontent.com/69894591/188416905-1a46074b-7c07-4c40-b1b1-2db2180b33b5.png">